### DISCUSSION TITLE
PC-Basic  Fedora

### DISCUSSION THREAD

**Discussion by grs77** [robhagemans/pcbasic#211] [General]
When trying to install python3-pcbasic-2.0.6-1.noarch.rpm  in fedora 36 I get messages of conflicts with a file from package filesystem-3.18-2.fc36.86_64.   In:
/usr/local/bin
/usr/local/lib
/usr/local/share
/usr/local/share
/usr/local/share/applications
/usr/localshare/man.
Any advice on how to fix please
GRS77

**Comment by robhagemans** [comment]
Hi, I think for people to be able to help you need to post a bit more information, could you perhaps copy & paste the literal error message you get? It's not clear to me what file from another package you could have a conflict with but perhaps someone who is familiar with Fedora has any suggestions.

**Comment by grs77** [comment]
I do not think I can add much but hopefully the screen shot provides some clues.   Not sure where to look to get perhaps lop info 
![Screenshot from 2022-08-18 09-21-00](https://user-images.githubusercontent.com/111403995/185260566-ea939ee7-89d6-4360-a0dd-1b799f10bf7d.png).
Procedure used.    Down load the Fedora rpm file. Click on it in the Downloads directory.  It opens the Software app (running gnome). click on install  and I get the message above.  
Hope it helps.  Thanks

**Comment by robhagemans** [comment]
This issue appears to be the one discussed and resolved here: 
https://stackoverflow.com/questions/27172142/conflicts-with-file-from-package-filesystem-3-2 
The fix provided in the link suggests a way to rebuild the package to remove the conflict.

I'll see if I can update the `.rpm` package so in future this won't be necessary.
